packages/headless/src/controllers/search-parameter-manager/headless-search-parameter-manager.ts

```diff
@@ -43,7 +46,7 @@ export type SearchParameterManager = ReturnType<
 export type SearchParameterManagerState = SearchParameterManager['state'];
 
 export function buildSearchParameterManager(
-  engine: Engine<Partial<SearchParametersState>>,
+  engine: Engine<Partial<SearchAppState>>,
```

Comment:
This interface change should not be necessary if we place the new property inside of the `advancedSearchQueries` slice.